src/modal/modal.component.ts

```diff
@@ -106,8 +109,28 @@ export class Modal implements AfterViewInit {
 
 	/**
 	 * Label for the modal.
+	 *
+	 * @deprecated since v4
 	 */
-	@Input() modalLabel = "default";
+	@Input() set modalLabel(value: string) {
+		this.ariaLabel = value;
+	}
+
+	get modalLabel() {
+		return this.ariaLabel;
+	}
+
+	@Input() ariaLabel = "default";
+
+	/**
+	 * Controls the visibility of the modal when used directly in a template
+	 */
+	@Input() open = false;
```

Comment:
The `In modal` story for `datepicker` needs to have the open property be set to true

Reply:
Not specifically to true, but I see what you mean 👍